templates/make_manifest

```diff
@@ -7,6 +7,7 @@ STEMCELL_OS=${STEMCELL_OS:-ubuntu}
 
 infrastructure=$1
 servers=$2
+cf_manifest=$3
```

Comment:
I added 'consul_agent' and 'metron_agent' to 'jobs.public_ha_proxy_z1.templates' because I thought 'consul_agent' and 'metron_agent' jobs are needed. So I needed the properties of the cf manifest. But sadly, I added two jobs and it did not deployed normally.
Sorry for the confusion. I'll try to verify it again.